releases/0.2.17/CHANGELOG_EXCERPT.md

```
@@ -0,0 +1,20 @@
+# EVOSEAL 0.2.17 - Release Notes
+
+*Released on: 2025-07-23*
```

markdownlint-cli2 (0.17.2)

Emphasis used instead of a heading

(MD036, no-emphasis-as-heading)